data/helpers.d/setting

```diff
@@ -274,7 +275,18 @@ ynh_permission_create() {
         allowed=",allowed=['${allowed//';'/"','"}']"
```

Comment:
Should I move this line in the else part line `287`?

Reply:
Yes